STUDIO/org.openl.rules.webstudio/src/org/openl/rules/webstudio/web/admin/GitRepositorySettings.java

```diff
@@ -35,13 +32,10 @@ public class GitRepositorySettings extends RepositorySettings {
     private final static String MAX_AUTHENTICATION_ATTEMPTS_SUFFIX = ".max-authentication-attempts";
     private final static String PROTECTED_BRANCHES_SUFFIX = ".protected-branches";
 
-    @Parameter(description = "Remote repository")
-    @JsonView(Views.Base.class)
-    private boolean remoteRepository;
-
     @Parameter(description = "URL")
     @SettingPropertyName(suffix = URI_SUFFIX)
     @JsonView(Views.Base.class)
+    @NotBlank
     private String uri;
```

Comment:
If `uri` accepts a local path in the same way as the previous l`ocalRepositoryPath`, similar validation logic should be applied.